### DISCUSSION TITLE
Globe sky color

### DISCUSSION THREAD

**Discussion by slaviczavik** [maplibre/maplibre-gl-js#5223] [General]
Hello,

I know the Globe is not stable yet, but I would like to ask if there is a plan for fix or implementation of [sky](https://maplibre.org/maplibre-style-spec/sky/) for the globe projection.

I was playing with `5.0.0-pre.9` and looks like the sky specification is working only for mercator projection. There is some sky default for globe projection, see pictures:

overview:
<img width="740" alt="Screenshot 2024-12-16 at 19 22 58" src="https://github.com/user-attachments/assets/2fd727f7-4de5-443c-81c2-95cec16caf23" />

detail (crop):
<img width="800" alt="Screenshot 2024-12-16 at 19 22 58 copy" src="https://github.com/user-attachments/assets/8f91e356-4efd-48da-832c-28f3065285e0" />

Is this an intentional behavior or is it a bug?

**Comment by HarelM** [comment]
Sly is not well supported for globe, your are welcome to contribute a fix. 
Having said that, this is not gating to the release of globe.
